src/libcore/num/mod.rs

```diff
@@ -44,7 +44,9 @@ pub struct Wrapping<T>(#[stable(feature = "rust1", since = "1.0.0")] pub T);
 pub mod wrapping;
 
 // All these modules are technically private and only exposed for libcoretest:
+#[cfg_attr(not(stage0), cfg(target_float))]
 pub mod flt2dec;
+#[cfg_attr(not(stage0), cfg(target_float))]
 pub mod dec2flt;
 pub mod bignum;
 pub mod diy_float;
```

Comment:
If flt2dec and dec2flt are cfg'd out, then bignum and diy_float can also be cfg'd out..